.github/workflows/test.yml

```diff
@@ -64,4 +87,5 @@ jobs:
             python -m pip install --upgrade pip
             pip install tox tox-gh-actions
         - name: Run twine-check
```

Comment:
I feel like if the twine job could have a condition on just this step, the job would still always execute, but only call tox if changedfiles.outputs.rst == False.  I am not sure whether if: can be used at that level, I have not yet found the right spot in the github actions documentation, it is just an idea.  While this check as written is clever, I am still not sure this is worth this level of effort at the current time - the job executes very quickly, so it is not slowing things down much to always run it.